### DISCUSSION TITLE
FluentTerminal

### DISCUSSION THREAD

**Discussion by mrdoge515** [catppuccin/catppuccin#1916] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

[FluentTerminal](https://github.com/felixse/FluentTerminal) can be themed using the `Themes` tab in settings.

### Has the requested port been themed already?

https://github.com/mrdoge515/FluentTerminal

**Comment by sgoudham** [comment]
Transferred to #1918
